tests/api/api/test_operations.py

```diff
@@ -59,13 +71,13 @@ def _mock_waiting_for_migration():
 
 
 def test_migrations_success(
-    # db calls init_data with from_scratch=True which means it will anyways do the migrations
-    # therefore in order to make the api to be started as if its in a state where migrations are needed
-    # we just add a middle fixture that sets the state
     db: sqlalchemy.orm.Session,
     _mock_waiting_for_migration,
     client: fastapi.testclient.TestClient,
 ) -> None:
+    # db calls init_data with from_scratch=True which means it will anyways do the migrations
+    # therefore in order to make the api to be started as if its in a state where migrations are needed
+    # we just add a middle fixture that sets the state
```

Comment:
Comments should be above the code they are referring to, in this case I"m explaining what is this weird `_mock_waiting_for_migration` fixture and that the declaration order matters, therefore I put the comment there